sn_node/tests/data_with_churn.rs

```diff
@@ -315,21 +320,40 @@ fn store_chunks_task(
         loop {
             let random_data = gen_random_data(*DATA_SIZE);
 
-            let data_map = client
-                .data_put(random_data, &wallet)
-                .await
-                .inspect_err(|err| {
-                    println!("Error to put chunk: {err:?}");
-                    error!("Error to put chunk: {err:?}")
-                })?;
-
-            println!("Stored Chunk/s at {data_map:?} after a delay of: {delay:?}");
-            info!("Stored Chunk/s at {data_map:?} after a delay of: {delay:?}");
+            // FIXME: The client does not have the retry repay to different payee feature yet.
```

Comment:
In that case, then a retry is no use?

Reply:
This retry is just to make sure that we pick a different payee if the original one goes offline. This will do the job for us now.